Dockerfile

```diff
@@ -1,9 +1,6 @@
-FROM fedora:20
+FROM centos:7
 MAINTAINER Marek Goldmann <mgoldman@redhat.com>
 
-# Execute system update
-RUN yum -y update && yum clean all
```

Comment:
You should rely on the base image being up to date. Doing a yum update in subsequent layers negates the point of using the base image. Where do you stop? Every image built from this one should do yum update too just in case? Layer upon layer of yum updates ad infinitum :) As well as hoping the  base image is a set of "known good" installed RPMs with fixed CVEs, etc.